test/rules/assertions/checkIndentation.js

```diff
@@ -48,6 +48,27 @@ export default {
         },
       ],
     },
+    {
+      code: `
+          /**
+           * foo
+           *
+           * @example
+           * anArray.filter((a) => {
+           *   return a.b;
+           * });
+           */
+          function quux () {
+
+          }
+      `,
+      errors: [
+        {
+          line: 7,
+          message: 'There must be no indentation.',
+        },
+      ],
+    },
```

Comment:
Can you add a test to make sure blocks after example block are still checked?
```js
/**
 * @example 
 * @returns
 *     eeee
 */
```

Reply:
I tested it locally, but yeah, it's a good idea to keep it.